<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PLCDeviceRunDialog</class>
 <widget class="QWidget" name="PLCDeviceRunDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PLC设备运行</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>边缘电机3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>边缘电机4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>传动带电机</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="frontTriggerPosEdit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>70</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>原点指示</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>错误状态</string>
   </property>
  </widget>
  <widget class="QComboBox" name="beltVecComboBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>30</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="forwardBtn">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>180</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>通讯指示</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>错误码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>传送点自动速度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>正面触发位置</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edgeTriggerPosEdit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>110</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>侧面触发位置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>180</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>进料传送带：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopBtn">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>180</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="CircleLabel" name="originLabel3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxx</string>
   </property>
  </widget>
  <widget class="CircleLabel" name="originLabel4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxx</string>
   </property>
  </widget>
  <widget class="CircleLabel" name="errStatusLabel3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxx</string>
   </property>
  </widget>
  <widget class="CircleLabel" name="errStatusLabelBelt">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxx</string>
   </property>
  </widget>
  <widget class="CircleLabel" name="errStatusLabelSystem">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxx</string>
   </property>
  </widget>
  <widget class="CircleLabel" name="errStatusLabelCon">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxx</string>
   </property>
  </widget>
  <widget class="CircleLabel" name="errStatusLabel4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxx</string>
   </property>
  </widget>
  <widget class="QLabel" name="errCodeLabelBelt">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="errCodeLabel3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="errCodeLabel4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CircleLabel</class>
   <extends>QLabel</extends>
   <header>circle_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
